### PR TITLE
added delete functionality

### PR DESCRIPTION
**Pull request by raeeschachar**

### CODE REVIEW

**Comment by raeeschachar** [comment]
@marykhan

File: contacts/models.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 from django.db import models
-from django.core.validators import RegexValidator
+from django.core.validators import RegexValidator, EmailValidator
```
Comment (MaryKhan) [inline contacts/models.py]: remove this import `EmailValidator`. :P

File: contacts/urls.py [hunk 2]
```diff
@@ -12,5 +12,6 @@
     url(r'^update_contact/(?P<pk>[0-9]+)/?$', views.UpdateContactView.as_view(), name='update_contact'),
     url(r'^update_contact_address/(?P<pk>[0-9]+)/?$',
         views.UpdateContactAddressView.as_view(), name='update_contact_address'
-        )
+        ),
+    url(r'^delete/(?P<pk>\d+)$', views.DeleteContactView.as_view(), name='delete_contact')
```
Comment (MaryKhan) [inline contacts/urls.py]: optional slash please?

**Review by MaryKhan — CHANGES_REQUESTED** [review]

**Comment by raeeschachar** [comment]
@MaryKhan Done.

**Pull request merged**